administrator/components/com_admin/script.php

```diff
@@ -2300,10 +2300,16 @@ private function migrateTinymceConfiguration(): bool
                  * "formats"      -> "styles"
                  * "template"     -> "jtemplate"
                  */
-                $params['configuration']['toolbars'][$setIdx]['menu'] = str_replace(
-                    ['blockformats', 'fontformats', 'fontsizes', 'formats', 'template'],
-                    ['blocks', 'fontfamily', 'fontsize', 'styles', 'jtemplate'],
-                    $toolbarConfig['menu']
+                $search  = ['blockformats', 'fontformats', 'fontsizes', 'formats'];
+                $replace = ['blocks', 'fontfamily', 'fontsize', 'styles'];
+
+                // Don't redo the template
+                if (!str_contains($params['configuration']['toolbars'][$setIdx]['menu'], 'jtemplate')) {
+                    $search[]  = 'template';
+                    $replace[] = 'jtemplate';
+                }
+
+                $params['configuration']['toolbars'][$setIdx]['menu'] = str_replace($search, $replace, $toolbarConfig['menu']
                 );
```

Comment:
```suggestion
                $params['configuration']['toolbars'][$setIdx]['menu'] = str_replace($search, $replace, $toolbarConfig['menu']);
```
drone fail (phpcs) so no download package is available